package.json

```diff
@@ -7,17 +7,28 @@
   "scripts": {
     "test": "wdio run src/configs/wdio.conf.js",
     "wdio": "wdio run ./wdio.conf.js",
-    "test:api": "npx newman run postman/TrelloApiBoard.collection.json --environment postman/TrelloApiBoard.environment.json"
+    "test:api": "npx newman run postman/TrelloApiBoard.collection.json --environment postman/TrelloApiBoard.environment.json",
+    "format": "prettier --write .",
+    "lint": "eslint .",
+    "lint:fix": "eslint . --fix",
+    "styletest": "npm-run-all format lint wdio test:api"
```

Comment:
you do not need style code before every test run. You need to do this before commit